### PR TITLE
[th2-2715] added util methods from store-common to use in estore/mstore

### PR DESCRIPTION
**Pull request by andreydrobynin**

### CODE REVIEW

**Comment by andreydrobynin** [comment]
Related to https://github.com/th2-net/th2-estore/pull/25

File: src/main/kotlin/com/exactpro/th2/common/util/StorageUtils.kt [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ */
+
+@file:JvmName("StorageUtils")
+
+package com.exactpro.th2.common.util
+
+import com.exactpro.th2.common.grpc.Direction
+import com.google.protobuf.TimestampOrBuilder
+import java.time.Instant
+
```
Comment (Nikita-Smirnov-Exactpro) [inline src/main/kotlin/com/exactpro/th2/common/util/StorageUtils.kt:24]: Please add 
FIXME comment: this code should be extracted to a separate library together with cradle extraction.

File: gradle.properties [hunk 2]
```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 #
 
-release_version=3.29.2
+release_version=3.29.3
```
Comment (Nikita-Smirnov-Exactpro) [inline gradle.properties:16]: Please update the minor version according API has been extended

**Review by Nikita-Smirnov-Exactpro — CHANGES_REQUESTED** [review]

File: src/main/kotlin/com/exactpro/th2/common/util/StorageUtils.kt (hunk 1 above)
Comment (andreydrobynin) [reply src/main/kotlin/com/exactpro/th2/common/util/StorageUtils.kt:24]: done

File: gradle.properties (hunk 2 above)
Comment (andreydrobynin) [reply gradle.properties]: done

**Review by Nikita-Smirnov-Exactpro — APPROVED** [review]

**Review by Nikita-Smirnov-Exactpro — APPROVED** [review]

**Pull request merged**